src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingDataMap.js

```diff
@@ -181,130 +181,74 @@ export const BenchmarkingDataMap = {
             },
         ],
     },
-    Lung: {
-        navBarTitle: 'Lung',
-        title: 'Lung Primary Tissue Data',
+    Donor1: {
+        navBarTitle: 'Donor ST001',
+        title: 'Donor ST001 : Benchmarking Tissue Data',
         description: primaryTissuePageDescription,
         type: 'Primary Tissue Data',
-        path: '/data/benchmarking/lung',
+        path: '/data/benchmarking/donor-st001',
         tabMapArray: [
             {
-                eventKey: '#donor-1',
-                title: 'Donor 1',
-                // searchHref: '/search/?type=File&status=released&status=restricted&status=public',
+                eventKey: '#liver',
+                title: 'Liver',
+                searchHref:
+                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analyte.samples.sample_sources.code=ST001-1A',
```

Comment:
I've merged my branch, so the update below is required for all these searches:

```suggestion
                    '/search/?type=File&status=released&status=restricted&status=public&file_sets.libraries.analytes.samples.sample_sources.code=ST001-1A',
```

Reply:
Thanks for this reminder. Went ahead and made that change.